### ISSUE TITLE
ALLOW *.tommi.space / remove visits.tommi.space from the list

### ISSUE THREAD

**Issue by xplosionmind** [easylist/easylist#15069]
### List the website(s) you're having issues:

- `https://visits.tommi.space`
- `*.tommi.space`

### What happens?

The domain is blocked and it should not be… it is not a tracking source!

### List Subscriptions you're using:

All the default uBlock Origin ones

### Your settings

uBlock origin alone with default settings

- OS/version: MacOS 13.1
- Browser/version: Firefox 110.0
- Adblock Extension/version: 1.47.0

### Other details:

See https://forums.lanik.us/viewtopic.php?t=47997-visits-tommi-space

**Comment by iam-py-test** [comment]
Looking at `https://visits.tommi.space/umami.js`, this seems to fall under:
> Impressions / Event / Perf / Pageview logging

In [the EasyPrivacy policy](https://github.com/easylist/easylist#easyprivacy), which ryanbr/fanboy linked to on the original forum post.
This script appears to collect the current URL, browser language, and screen width and height, and sends that data to the server.
**Note: I am not part of the EasyList team.**

**Comment by xplosionmind** [comment]
> This script appears to collect the current URL, browser language, and screen width and height, and sends that data to the server.

I could retrieve most of this data by accessing the server logs. I am sorry I am a bit confused by the many lists and directives… I understend if my personal analytics are still excluded.

**Comment by iam-py-test** [comment]
> it is not a tracking source
> my personal analytics

??
> I could retrieve most of this data by accessing the server logs.

Screen width and height (as far as I am aware) isn't sent in the headers. But I understand your point.

My opinion does not matter, as I am not part of the EasyList team and my opinion holds no weight here. However, based on my interpretation of the EasyList rules, that rule is valid.

**Comment by Khrin** [comment]
> The domain is blocked and it should not be… it is not a tracking source!

@xplosionmind Sei italiano, giusto? EasyPrivacy serve anche a bloccare script come `https://visits.tommi.space/umami.js`  quindi direi che il filtro rimane.

**Comment by xplosionmind** [comment]
Ciao @Khrin and hi to all the kind people that read this issue.

I was wondering if by removing from `https://visits.tommi.space/umami.js` the incriminated non-EasyPrivacy-compliant lines the domain could be unblocked.

I am sorry, but I am pretty clueless about JavaScript… the [content of the JavaScript analytics file `umami.js` is here](https://paste.yunohost.org/refeciraqi.js), could you tell me which lines I should be removing in order to make this script acceptable by EasyList?

**Comment by xplosionmind** [comment]
I posted this problem also in https://github.com/umami-software/umami/issues/1835
